crowdin/messages.json

```diff
@@ -1582,7 +1583,6 @@
   "2126425899": "Adidas Salomon",
   "2127438954": "For synthetic, the swap charge is calculated on an annual basis for long and short positions based on this formula:",
   "2130006600": "Deriv P2P is a flawless innovation.",
-  "2130695061": "Cookies help us to give you a better experience and personalised content on our site. ",
```

Comment:
This may not translate, please check in your local and make sure about it :pray: